optimize/example_run.py

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env python3
+
+from ParamFitter import ParamFitter
+
+def main():
+    param_fit = ParamFitter(['eField', 'lifetime'], lr=1e1) 
+    param_fit.make_target_sim(seed=2)
+    param_fit.load_data(filename='/sdf/group/neutrino/cyifan/muon-sim/fake_data_S1/edepsim-output.h5')
```

Comment:
How about adding command line arguments for the parameter list, learning rate, and data file path?

Reply:
Yup yup, will do! Sorry -- this was in the plans, but merging needed for batching, so things might be a little half baked

Reply:
Done! In case we want to use different learning rates for different parameters, doing like a yaml config might be good, but this can work for now